### ISSUE TITLE
Neosnippet not called when autocomplations are open

### ISSUE THREAD

**Issue by otavioschwanck** [wsdjeg/SpaceVim#2658]
## Expected behavior
press tab and use the snippet.

## The reproduce ways from Vim starting (Required!)

Open a rb file

type `def`, wait a second, press tab

## Screenshots

![neo snippet](https://user-images.githubusercontent.com/9551316/54177856-7ae06e80-4472-11e9-8e21-acfcd626975e.gif)

**Comment by Shougo** [comment]
I think `g:neosnippet_complete_done` is enabled.

```
					*g:neosnippet#enable_complete_done*
g:neosnippet#enable_complete_done
		If this variable is not 0, neosnippet can expand on
		|CompleteDone|.
		Note: If the feature is enabled, you cannot expand snippet
		trigger manually when |pumvisible()|to prevent conflicts with
		|CompleteDone| expand.

		The default value is 0.
```

**Comment by otavioschwanck** [comment]
> I think `g:neosnippet_complete_done` is enabled.
> 
> ```
> 					*g:neosnippet#enable_complete_done*
> g:neosnippet#enable_complete_done
> 		If this variable is not 0, neosnippet can expand on
> 		|CompleteDone|.
> 		Note: If the feature is enabled, you cannot expand snippet
> 		trigger manually when |pumvisible()|to prevent conflicts with
> 		|CompleteDone| expand.
> 
> 		The default value is 0.
> ```

How can i fix this? its a little annoying

**Comment by Shougo** [comment]
It cannot be fixed.
Because, it is disabled by conflict.

You can disable the option.

**Comment by wsdjeg** [comment]
will be fixed by #2674
